setup.cfg

```diff
@@ -18,40 +18,27 @@ search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
 
 [flake8]
-exclude = 
+exclude =
 	.git,
 	docs,
 	build,
 	.eggs,
 	tests
-max-line-length = 88
-max-complexity = 12
```

Comment:
is removing this related to "ruff" or some independent change?

Reply:
Yeah, this part is weird: so we had a max-line-length set to 88 (which is needed to conform with black) but we regularly broke that rule all over the place. I opted for us to adopt 150 (black formatting will try to aim for 88 all the same), where lines longer than that are considered failures (or they're explicitly ignored, like in the JSON translations)

The Ruff equivalent tool for looking at complexity is called `mccabe`. This configuration was also moved to the `pyproject.toml` file.

Reply:
Ah ok, "flake" needs to do less things that are being done by "ruff" now, correct?